### PR TITLE
README: Add badges

### PR DESCRIPTION
**Pull request by matevz**

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-name: Test deployment
```
Comment (tjanez) [inline .github/workflows/ci-test.yml:1]: Can you add a comment similar to Oasis Core's ci-lint.yml:
```
# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
```

File: .github/workflows/ci-test.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 on:
   pull_request:
     branches: [main]
 
 jobs:
-  test-deploy:
-    name: Test deployment
+  test-build:
+    name: Test build
```
Comment (tjanez) [inline .github/workflows/ci-test.yml]: Can you add a comment similar to Oasis Core's ci-lint.yml:
```
# NOTE: This name appears in GitHub's Checks API.
```

Comment (tjanez) [inline .github/workflows/ci-test.yml]: Can you use a naming scheme similar to Oasis Core?
E.g. name this `ci-test`?

File: .github/workflows/ci-test.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
-name: Test deployment
+name: Tests
```
Comment (tjanez) [inline .github/workflows/ci-test.yml]: Can you use a naming scheme similar to Oasis Core?
E.g. name this `ci-test`?

File: .github/workflows/deploy-main.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
-name: Deploy to GitHub Pages
+name: Deployment
```
Comment (tjanez) [inline .github/workflows/deploy-main.yml]: Can you use a naming scheme similar to Oasis Core?
E.g. name this `deploy` or `deployment`?

File: .github/workflows/deploy-main.yml [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 
 on:
   push:
     branches: [main]
 
 jobs:
   deploy:
-    name: Deploy to GitHub Pages
+    name: Deployment
```
Comment (tjanez) [inline .github/workflows/deploy-main.yml]: Same here.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,9 @@
 # Oasis Docs
 
+![Tests](https://github.com/oasisprotocol/docs/actions/workflows/ci-test.yml/badge.svg)
```
Comment (tjanez) [inline README.md]: Same here, adjust according to above.

**Review by tjanez — COMMENTED** [review]
Please, also rename appropriately the ci-links.yml workflow

File: README.md (hunk 6 above)
Comment (matevz) [reply README.md]: Adjusted according to https://github.com/oasisprotocol/oasis-core/blob/master/README.md?plain=1#L3-L5

**Review by tjanez — APPROVED** [review]

**Pull request merged**